packages/nimble-components/src/rich-text-editor/specs/README.md

```diff
@@ -320,20 +335,69 @@ markdown based on [CommonMark](http://commonmark.org/) flavor:
 -   Italics - `*Italics*`
 -   Numbered list - `1. Numbered list`
 -   Bulleted list - `* Bulleted list`
+-   Absolute URL links - `<Absolute URI link>` (For more details on the markdown syntax for absolute URL links, see [Autolinks in CommonMark](https://spec.commonmark.org/0.30/#autolink))
+
+_Configurations on Tiptap to support only absolute links_:
+
+Install the [link extension](https://tiptap.dev/api/marks/link) mark from Tiptap and initialize the `Links` with the following configurations:
+
+1.  Set regular expression in [validate](https://tiptap.dev/api/marks/link#validate) field to support only `HTTP` and `HTTPS` absolute links in the editor.
+2.  Set [openOnClick](https://tiptap.dev/api/marks/link#open-on-click) to `false` for the editor, to restrict the user from opening a link from the editor by clicking. Users can open the link only by
+    `Right-click >> Open link in new tab` from the editor.
+3.  Set [autoLink](https://tiptap.dev/api/marks/link#autolink) to `true`, to add the valid link automatically when typing.
+4.  Set [linkOnPaste](https://tiptap.dev/api/marks/link#link-on-paste) to `false` which will attach the URL to current selected text in the editor, converting it into a hyperlink. If it is `true`,
+    pasting a link to the selection will add the link behind the word which is not supported for the initial pass.
 
 The `nimble-rich-text-viewer` will be responsible for converting the input markdown string to HTML Fragments with the help of
 `prosemirror-markdown` parser, which is then converted to HTML string and rendered into the component to view all rich text content.
 
+_Implementation details for supporting absolute link:_
+
+For the `nimble-rich-text-viewer` component, we will set up the `link` mark in the Prosemirror schema as below, allowing links in the component to open with default behavior (same tab).
+
+Only links with schema HTTP and HTTPS will be treated as links within `nimble-rich-text-editor` and `nimble-rich-text-viewer` and rest of the links with any other schemas will be rendered as plain text.
+
+Here is the default [link configuration](https://github.com/ProseMirror/prosemirror-markdown/blob/b7c1fd2fb74c7564bfe5428c7c8141ded7ebdd9f/src/schema.ts#L138C5-L148C6)
+from the `prosemirror-markdown` package for comparison with the newly updated configuration.
+
+```js
+link: {
+    attrs: {
+        href: {},
+        rel: { default: 'noopener noreferrer' }
+    },
+    inclusive: false,
+    toDOM(node) {
+        return [
+            anchorTag, // nimble-anchor here
+            {
+                href: node.attrs.href as Attr,
+                rel: node.attrs.rel as Attr
+            }
+        ];
+    }
+}
+```
+
+1.  We will set the `rel` attribute value to `noopener noreferrer` to enhance security and ensure responsible linking practices.
+2.  In the `toDOM` function, we have incorporated the `anchorTag` to render all links within the viewer component as `nimble-anchor` elements.
```

Comment:
One thing to double check (if you haven't already): does the nimble-anchor render as expected if the user applies bold or italic styling to it? I noticed the anchor matrix story has a test case for italic but I didn't see one for bold.

Reply:
On a quick check, looks like yes. nimble-anchor is able to support this rendering both in viewer and editor.

**Viewer**
![MicrosoftTeams-image](https://github.com/ni/nimble/assets/94439533/d8b6a3ce-4ef4-4552-806b-da6f4d2e598d)

**Editor**
![MicrosoftTeams-image (1)](https://github.com/ni/nimble/assets/94439533/bc265853-fade-4fbd-b51b-0b1645fac89f)

Reply:
Partial formatting also looks to work fine. `nimble-anchor` is awesome! 😃 

Will resolve this thread after getting your acknowledgement or further pre-cautionary inputs to be considered.

![MicrosoftTeams-image (2)](https://github.com/ni/nimble/assets/94439533/2d89c57f-60c6-48ec-a142-18e8a8292485)

Reply:
Great, thanks for checking. It'd be good to add these as test cases when you get to implementation.